### PR TITLE
Stubtest fixes

### PR DESCRIPTION
**Pull request by hatal175**

### CODE REVIEW

**Comment by hatal175** [comment]
I see that @hauntsaninja made a more extensive path change. Merge his before this and I'll fix the issues.

File: stdlib/socket.pyi [hunk 1]
```diff
@@ -768,7 +768,13 @@ def getnameinfo(sockaddr: Union[Tuple[str, int], Tuple[str, int, int, int]], fla
 def getprotobyname(protocolname: str) -> int: ...
 def getservbyname(servicename: str, protocolname: str = ...) -> int: ...
 def getservbyport(port: int, protocolname: str = ...) -> str: ...
-def socketpair(family: int = ..., type: int = ..., proto: int = ...) -> Tuple[socket, socket]: ...
+
+if sys.platform == "win32":
+    def socketpair(family: int = ..., type: int = ..., proto: int = ...) -> Tuple[socket, socket]: ...
```
Comment (Akuli) [inline stdlib/socket.pyi:773]: Is it possible to use `family: Literal[AF_INET, AF_INET6]`? I think `Literal` should support enums.

Comment (hatal175) [reply stdlib/socket.pyi:773]: The default value is AF_INET which is int, not AddressFamily, so that gives an error.

Comment (Akuli) [reply stdlib/socket.pyi:773]:
```
$ grep AF_INET: stdlib/socket.pyi
AF_INET: AddressFamily
        AF_INET: int
```

The indented one is inside an enum, meaning the value of the enum has type `int`, although the enum member doesn't.

**Comment by hatal175** [comment]
Ok, so the real function has an int as a default value.
In the stub, AF_INET is of type AddressFamily which is an enum above python
3.4.

So they are incompatible. At least from what I tried.

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**